src/gui/preview.ui

```diff
@@ -13,6 +13,9 @@
   <property name="windowTitle">
    <string>Preview selection</string>
   </property>
+  <property name="modal">
+   <bool>true</bool>
```

Comment:
Why did you make it modal?

Reply:
Because i think it's pure modal 😀, user should not have option to open main menu or change torrent in torrentwidgetlist when preview modal :) is opened.
User selects in context menu, that he wants to preview some torrent, preview modal is opened, his options are to select some file and preview or cancel.
I can not imagine a better example for modal dialog :), what do you think?

Reply:
User have two torrents in progress, downloading the same subject, and they want to compare quality of media files in both torrents to select only one.

Reply:
Of course, no problem to revert it :), just tell